conversionFunctions.f90

```diff
@@ -31,42 +31,26 @@ pure function calcM( press, temp ) result ( m )
     real(kind=DP), intent(in) :: press, temp
     real(kind=DP) :: m, press_pa
 
-    press_pa = press * 1.0d+2
-    m = ( 6.02214129d+23 / 8.3144621 ) * ( press_pa / temp )
+    press_pa = press * 1.0d+02
+    m = 1.0d-06 * ( 6.02214129d+23 / 8.3144621 ) * ( press_pa / temp )
     return
   end function calcM
 
   ! ----------------------------------------------------------------- !
 
-  pure function calcPressure( m, temp ) result ( press )
-    ! calculate pressure (mbar) from the number density of air
-    ! (molecule cm-3) and temperature (K)
-    implicit none
-
-    real(kind=DP), intent(in) :: m, temp
-    real(kind=DP) :: press
-    real(kind=DP) :: press_pa
-
-    press_pa = ( m * temp ) / ( 6.02214129d+23 / 8.3144621 )
-    press = press_pa/1.0d+2
-    return
-  end function calcPressure
-
-  ! ----------------------------------------------------------------- !
-
   pure function convertRHtoH2O( rh, temp, press ) result ( h2o )
     ! convert relative humidity to water concentration (molecule cm-3)
-    ! pressure in mbar, temperature in K
+    ! pressure in mbar (1 mbar = 1 hPa), temperature in K
     implicit none
 
     real(kind=DP), intent(in) :: rh, temp, press
-    real(kind=DP) :: h2o, exponent, e1
+    real(kind=DP) :: h2o, h2o_ppm, temp_c, wvp
 
-    exponent = exp( -1.00d00 * ( 597.30d00 - 0.57d00 * ( temp - 273.16d00 ) ) * &
-                   18.00d00 / 1.986d00 * ( 1.00d00 / temp - 1.00d00 / 273.16d00 ) )
+    temp_c = temp - 273.15
```

Comment:
Could you write a comment detailing where this calculation comes from, both the document and the equations? As far as I can tell, it looks like the combination of (6) and (1), then (18) in the Vaisala document. The conversion from h2o_ppm to h2o I also understand, but is there an explicit formula we can link to, or at least write in the comment?